python/src/PhysicalSpaceCrossEntropyImportanceSampling_doc.i.in

```diff
@@ -0,0 +1,134 @@
+and unknown.
+
+The Physical Space Cross-Entropy Importance Sampling algorithm [rubinstein2017]_ uses a parametric auxiliary distribution :math:`h_\mathbf{\lambda}` in order to optimize its parameters to compute the probability of interest with accuracy. It involves an auxiliary optimization problem to find the auxiliary distribution parameters :math:`\mathbf{\lambda}` minimizing the Kullback-Leibler divergence with respect to :math:`h_{opt}`. The following algorithm is used:
+
+1. :math:`k=1`, set :math:`\rho \in [0,1]` and :math:`h_0 = f_{\mathbf{X}}`
+
+2. Generate the population :math:`\mathbf{x}_1^{(k)},...,\mathbf{x}_N^{(k)}` according to the PDF :math:`h_{k-1}`, apply the 
+   function :math:`g` in order to have :math:`y_1^{(k)}=g(\mathbf{x}_1^{(k)}),...,y_N^{(k)} = g(\mathbf{x}_N^{(k)})`
+
+3. Compute the empirical :math:`\rho`-quantile :math:`q_k=\max(T,y^{(k)}_{\left \lfloor \rho N \right\rfloor})` using the floor of :math:`\rho N`.
+
+4. If :math:`q_k>T`: go to Step 7
+
+5. Estimate the auxiliary distribution parameters:
+
+   .. math::
+
+       \mathbf{\lambda}_{k}= \textrm{argmax}_{\lambda} \frac{1}{N} \sum_{i=1}^{N}{\mathbf{1}}_{g(\mathbf{x}_i^{(k)}) \leq q_k} \frac{f_\mathbf{X}(\mathbf{x}_i^{(k)})}{h_{\mathbf{\lambda}_{k-1}}(\mathbf{x}_i^{(k)})} \log(h_{\mathbf{\lambda}}(\mathbf{x}_i^{(k)}))
+
+6. :math:`k\leftarrow k+1`, go to Step 2
+
+7. Estimate the probability :math:`\widehat{P}^{CE}(g(\mathbf{\mathbf{X}})<T)=\frac{1}{N}\displaystyle \sum_{i=1}^{N} \mathbf{1}_{g(\mathbf{x}_i^{(k)})<T} \frac{f_\mathbf{X}(\mathbf{x}_i^{(k)})}{h_{\mathbf{\lambda}_{k-1}(\mathbf{x}_i^{(k)})}}`.
+
+
+Default solver for the optimization of auxiliary distribution is `LD_LBFGS` from :class:`~openturns.NLopt`.
+
+See also
+--------
+StandardSpaceCrossEntropyImportanceSampling, CrossEntropyImportanceSampling, CrossEntropyResult
+
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> ot.RandomGenerator.SetSeed(0)
+>>> distribution_R = ot.LogNormalMuSigma(300.0, 30.0, 0.0).getDistribution()
+>>> distribution_F = ot.Normal(75e3, 5e3)
+>>> marginals = [distribution_R, distribution_F]
+>>> distribution = ot.ComposedDistribution(marginals)
+>>> # create the model
+>>> model = ot.SymbolicFunction(['R', 'F'], ['R-F/(pi_*100.0)'])   
+>>> #create the event 
+>>> vect = ot.RandomVector(distribution)
+>>> G = ot.CompositeRandomVector(model, vect)
+>>> event = ot.ThresholdEvent(G, ot.Less(), 0.0)
+>>> distribution_marginR = ot.LogNormalMuSigma().getDistribution()
+>>> distribution_marginF = ot.Normal()
+>>> aux_marginals = [distribution_marginR, distribution_marginF]
+>>> aux_distribution = ot.ComposedDistribution(aux_marginals)
+>>> ## Definition of parameters to be optimized
+>>> active_parameters = ot.Indices([0,1,2,3,4]) # active parameters from the auxiliary distribution which will be optimized
```

Comment:
no need to specify Indices, a list of int should work

Reply:
Yes, you're right. fixed